test/views/HonestyDeclarationViewSpec.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import forms.HonestyDeclarationForm
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+
+class HonestyDeclarationViewSpec extends ViewBaseSpec {
+
+  "Rendering the honesty declaration page" when {
+
+    "the user is an agent" should {
```

Comment:
This first set of tests are for a principal user and the set below are for an agent - could you just swap the descriptions around to match this